genesyscloud/routing_email_route/resource_genesyscloud_routing_email_route_utils.go

```diff
@@ -26,7 +27,7 @@ func getRoutingEmailRouteFromResourceData(d *schema.ResourceData) platformclient
 		Skills:    util.BuildSdkDomainEntityRefArr(d, "skill_ids"),
 		Language:  util.BuildSdkDomainEntityRef(d, "language_id"),
 		FromName:  platformclientv2.String(d.Get("from_name").(string)),
-		FromEmail: platformclientv2.String(d.Get("from_email").(string)),
+		FromEmail: buildFromEmail(d),
```

Comment:
API doesn't like passing in FromEmail as an empty string. If its empty, just make it nil straight up so its not passed to the API.